backend/src/openarchiefbeheer/utils/tests/gherkin.py

```diff
@@ -470,6 +618,20 @@ async def inverted_method(*args, **kwargs):
 
             return InvertedThen(self)
 
+        async def archive_configuration_should_be(self, page, **kwargs):
+            # Add minor delay (0.1s) to allow the database to update.
+            await page.wait_for_timeout(100)
+
+            @sync_to_async()
+            def get_archive_config():
+                return ArchiveConfig.get_solo()
```

Comment:
We might need to clear the solo cache for tests that update the ArchiveConfig, otherwise we might run into test isolation issues